packages/content/test/attributes/Website.test.ts

```diff
@@ -0,0 +1,73 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { Website } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type Website", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type Website", function () {
+        const validWebsite = Website.from({
+            value: "https://inwind.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type Website with the German 'ä' in its domain", function () {
+        const validWebsite = Website.from({
+            value: "https://inwänd.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwänd.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type Website without www.", function () {
```

Comment:
You neither test websites starting with `www.` before. And in this test you omitted `https:`.

Reply:
okay I will do this before.